
try:
    raise Exception("dzwoń po policję")

except Exception:
    print("tutaj jeszcze nie error")
raise Exception("inny error")